stablehlo/testdata/conv_general_dilated_depthwise1d_dilated_lhs_float32_2_3_9__rhs_float32_12_1_3__windowstrides__1___p4726467912149865720.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN-DISABLED: stablehlo-opt -inline %s | stablehlo-translate --interpret
+// RUN: stablehlo-opt -inline %s | stablehlo-translate --interpret
```

Comment:
Can you validate that these work on windows / mac as well -- can use internal infra to validate. I thought we had FP issues with some numerics, are we safe to enable all these?

Reply:
I've verified that these tests successfully pass internal CI on all platforms. FP issue we had was due to a magic number we used for fp tolerance for all data types: [link](https://github.com/openxla/stablehlo/blob/b9118430c760e1283d2e48791e80065b7dcc0db5/stablehlo/reference/Element.cpp#L174)

Reply:
Also passes on GH CI except one. The failure seems to be unrelated to the change though: https://github.com/tensorflow/tensorflow/pull/66020.